### PR TITLE
Spec no 2

### PR DESCRIPTION
**Pull request by Czarrrna**

### CODE REVIEW

File: app/views/cakes/_form.html.haml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+= form_for @cake do |form|
+  %p= form.label("nazwa")
+  %p= form.text_field :name
+  = resource_errors_for(@cake, :name)
+  %p= form.select :kind, options_for_select([["wybierz", ""], "tasty", "non-gluten", "vegan"], @cake.kind)
+  = resource_errors_for(@cake, :kind)
+  %p= form.label("przepis")
+  %p= form.text_area :description
+  = resource_errors_for(@cake, :description)
+  %p= form.submit "Wyślij"
```
Comment (pjar) [inline app/views/cakes/_form.html.haml]: Try to keep with one style when passing attributes to methods - with or without parenthesises.

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -7,6 +7,16 @@
...
+    Logged in as <strong><%= current_user.email %></strong>.
+    <%= link_to 'Edit profile', edit_user_registration_path, :class => 'navbar-link' %> |
+    <%= link_to "Logout", destroy_user_session_path, method: :delete, :class => 'navbar-link'  %>
+  <% else %>
+    <%= link_to "Sign up", new_user_registration_path, :class => 'navbar-link'  %> |
+    <%= link_to "Login", new_user_session_path, :class => 'navbar-link'  %>
+  <% end %>
+
 <%= render "shared/flash" %>
 <%= yield %>
```
Comment (pjar) [inline app/views/layouts/application.html.erb]: You could change this to haml too

File: app/views/layouts/application.html.haml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+!!!
+%html
+  %head
+    %title CakeMaster
+    = stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true
+    = javascript_include_tag 'application', 'data-turbolinks-track' => true
+    = csrf_meta_tags
+  %body
+    = render "shared/flash"
+    = yield
```
Comment (pjar) [inline app/views/layouts/application.html.haml]: Please decide which application layout you use

File: config/routes.rb [hunk 4]
```diff
@@ -4,11 +4,12 @@
...
 
   resources :cake_attachments
+  devise_for :users
   # The priority is based upon order of creation: first created -> highest priority.
   # See how all your routes lay out with "rake routes".
 
   # You can have the root of your site routed with "root"
-  root 'cakes#index'
+  root to: 'cakes#index'
   resources :cakes
```
Comment (pjar) [inline config/routes.rb]: Please check if you use all CRUD actions for cakes, and change accordingly

File: app/views/cakes/_form.html.haml (hunk 1 above)
Comment (Czarrrna) [inline app/views/cakes/_form.html.haml]: done

File: app/views/layouts/application.html.erb (hunk 2 above)
Comment (Czarrrna) [inline app/views/layouts/application.html.erb]: done

File: config/routes.rb (hunk 4 above)
Comment (Czarrrna) [inline config/routes.rb]: done.

**Comment by pjar** [comment]
Please merge

**Pull request merged**